modules/reference/pages/rpk/rpk-registry/rpk-registry-mode-get.adoc

```diff
@@ -0,0 +1,34 @@
+= rpk registry mode get
+// tag::single-source[]
+
+Get Schema registry's mode.
+
+Running this command with no subject returns the global mode, alternatively you can use the `--global` flag to get the global mode at the same time as per-subject modes.
```

Comment:
```suggestion
Running this command with no subject returns the global mode for all schemas managed by Schema Registry. Alternatively, use the `--global` flag to get the global mode at the same time as per-subject modes.
```

Reply:
You can set a mode either globally, or on a per-subject basis. The Global mode is like a default, so, if a schema does not have a per-subject mode, it will default to the global mode.

Saying: ` for all schemas managed by Schema Registry.` maybe imply incorrectly that the global mode overrides the per-subject mode, what do you think?

Reply:
All schemas are managed by SR, no?

Reply:
```suggestion
Running this command with no subject returns Schema Registry's global mode. Alternatively, use the `--global` flag to return the global mode at the same time as per-subject modes.
```

Reply:
@Deflaimun, I've updated both descriptions closer to the original as agree with @r-vasquez that my original change might be misleading.